README.md

```diff
@@ -30,6 +30,5 @@ This repository contains all submitted Centriuge Proposals. The gensis proposal
 |CP | Short title |
 |---|-------------|
 |[CP39](./cps/CP39.md) | EVM compatibility on the Centrifuge Chain and Centrifuge App |
```

Comment:
CP39 can also go to the Accepted CPs list.

Reply:
done